settlements/README.md

```diff
@@ -22,17 +22,21 @@ Document version 2.2.7.
 # Settlement flow
 
 Settlements are done as quickly as possible, and depend on banks.
-The current solution is as fast as technically possible.
+
 
 The settlement flow is as follows:
 
 1. Day 1: A customer makes a purchase and the transaction is completed.
-   If the purchased product is shipped later, the "day 1" is the day the
+   Since a merchant should not capture the amount, i.e. charge the customer, until the purchased product is shipped, the "day 1" is the day the
    product is shipped and the customer's account is charged.
 2. Day 2: Settlement files are distributed, and are available on
    [portal.vipps.no](https://portal.vipps.no).
-3. Day 3 (the next _bank day_) at 16:00: Payments are made from Vipps' bank account to the merchant's bank account.
-4. Day 5 (the third _bank day_): The settlement is booked, with reference, by the bank.
+3. Day 3: Payments are made from Vipps' bank account to the merchant's bank account.
+4. Day 5: The settlement is booked, with reference, by the bank.
```

Comment:
This doesn't make sense. There is no "Day 5" 
If you want to mention "booked" it should be under Day 3. 
Happy flow: If Vipps pays me as a merchant on 01-12-21 before cutoff at the bank (usualy around 14:30), the money will reach the merchant bank account also on the 01-12-21. And when the merchant gets his bank statement from his bank midnight or night until the 02-12-21 he will see that value date and booking date on the settlement payment from Vipps are both 01-12-21.

Please also note that what we promise our merchants in our T&C's are the following that they will get the settlement at the latest day after we have received the funds. What we have described in the doumentation is how it usually works when things go well. Meaning merchants cannot come and complain if they get it according to what we say T&C. Could be that we should refer to T&C in the documentation and then say that what is described in the documentation is how it will usually work?
https://www.vipps.no/vilkar/vilkar-bedrift/
"5.2
Vipps utbetaler netto-oppgjør av gjennomførte Vipps-transaksjoner til Brukerstedets oppgjørskonto i Vipps innen en virkedag etter at midlene er kommet frem til Vipps, med mindre en annen oppgjørstid eller –metode er avtalt mellom Brukerstedet og Vipps. Vipps er ansvarlig for gjennomføring av betalingen inntil den er kommet frem til denne bedriftens bank. Når Vipps har gjennomført betaling av oppgjør til det kontonummer som er oppgitt som oppgjørskonto av Brukerstedet, skal oppgjøret anses å være foretatt til korrekt betalingsmottaker. "

"5.4
Vipps kan benytte en innløser for Vipps-transaksjoner som baserer seg på sluttbrukerens betalingskort. Overføringstiden mellom sluttbruker og Brukerstedet avhenger i slike tilfeller av overføringstiden mellom innløser og Vipps. Vipps vil normalt overføre midler til Brukerstedets oppgjørskonto innen neste virkedag etter at Vipps har mottatt midler fra innløser."

Reply:
updated in a new commit